docs/manage-security-verification-for-wog-account.md

```diff
@@ -0,0 +1,33 @@
+# Set up security verification for Whole of Government (WOG) account
+
+We know that public officers can use their TechPass account to access Singapore Tech Stack (SGTS) services and tools. However, if public officers want to access SGTS services and tools from their Government Managed Device (GMD) using TechPass, they must first set up security verification for their WOG account. This adds an additional level of security layer when they access these resources from their GMD.
+
+To achieve this, public officers set up a multi-factor authentication (MFA) for their WOG account using their Government Standard Imaging Build (GSIB) device.
```

Comment:
"To achieve this, public officers need to set up a mul"

Reply:
done